practice_app/pages/api/getRedditDataModule.js

```diff
@@ -0,0 +1,9 @@
+exports.getPostData = function (response) {
+    const values = response.data.children
+    var outJSON = []
```

Comment:
Not a big deal but, I believe using `var` keyword is outdated when declaring variables. You should use `let` and  instead.